tests/unittest/coordinator_rqid/main_test.go

```diff
@@ -75,32 +75,33 @@ func TestCoordinatorRqId(t *testing.T) {
 	db.SetMaxIdleConns(0)
 	defer db.Close()
 
-	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
 	// cleanup and insert one row in the table
 	conn, err := db.Conn(ctx)
 	if err != nil {
-		t.Fatalf("Error getting connection %s\n", err.Error())
+		t.Errorf("Error getting connection %s\n", err.Error())
 	}
 	tx, _ := conn.BeginTx(ctx, nil)
 	stmt, _ := tx.PrepareContext(ctx, "/*TestCoordinatorRqId*/delete from "+tableName)
 	_, err = stmt.Exec()
 	if err != nil {
-		t.Fatalf("Error preparing test (delete table) %s\n", err.Error())
+		t.Errorf("Error preparing test (delete table) %s\n", err.Error())
 	}
 	stmt, _ = tx.PrepareContext(ctx, "/*TestCoordinatorRqId*/insert into "+tableName+" (id, int_val, str_val) VALUES(?, ?, ?)")
 	_, err = stmt.Exec(1, time.Now().Unix(), "val 1")
 	if err != nil {
-		t.Fatalf("Error preparing test (create row in table) %s\n", err.Error())
+		t.Errorf("Error preparing test (create row in table) %s\n", err.Error())
 	}
 	err = tx.Commit()
 	if err != nil {
-		t.Fatalf("Error commit %s\n", err.Error())
+		t.Errorf("Error commit %s\n", err.Error())
 	}
 
+	conn, err = db.Conn(ctx)
 	stmt, _ = conn.PrepareContext(ctx, "/*TestCoordinatorRqId*/Select id, int_val from "+tableName+" where id=?")
 	rows, _ := stmt.Query(1)
 	if !rows.Next() {
-		t.Fatalf("Expected 1 row")
+		t.Errorf("Expected 1 row")
```

Comment:
I've added code coverage which will probably break. I'm ok with pressing resolve button, merging, and fixing afterward. Perhaps the smoothest might be rebasing your branch and fixing. Feel free to kick it over to me

Reply:
Done